src/Magnum/Math/Quaternion.h

```diff
@@ -733,6 +746,39 @@ template<class T> Matrix3x3<T> Quaternion<T>::toMatrix() const {
     };
 }
 
+/* Algorithm from:
+   https://en.wikipedia.org/wiki/Conversion_between_quaternions_and_Euler_angles#Quaternion_to_Euler_Angles_Conversion */
+template<class T> Vector3<Rad<T>> Quaternion<T>::toEuler() const {
+    CORRADE_ASSERT(isNormalized(),
+        "Math::Quaternion::toEuler():" << *this << "is not normalized", {});
+
+    Vector3<Rad<T>> euler{NoInit};
+
+    T qx = vector().x();
+    T qy = vector().y();
+    T qz = vector().z();
+    T qw = scalar();
+
+    /* Roll (x-axis rotation) */
+    T sinrCosp = T(2.0) * (qw * qx + qy * qz);
+    T cosrCosp = T(1.0) - T(2.0) * (qx * qx + qy * qy);
+    euler.x() = Rad<T>(std::atan2(sinrCosp, cosrCosp));
+
+    /* Pitch (y-axis rotation) */
+    T sinp = T(2.0) * (qw * qy - qz * qx);
+    if(std::abs(sinp) >= T(1.0))
+        euler.y() = Rad<T>(std::copysign(Constants<T>::pi() / T(2.0), sinp)); /* Use 90 degrees if out of range */
```

Comment:
```suggestion
        euler.y() = Rad<T>(std::copysign(Constants<T>::piHalf(), sinp)); /* Use 90 degrees if out of range */
```

Also, codecov is reporting this line is not covered. Can you craft a value for the test that hits this condition? :) Then it'll be perfect.